### DISCUSSION TITLE
Read & Apply ReplayGain? (maybe even generating & writing?)

### DISCUSSION THREAD

**Discussion by hasezoey** [tramhao/termusic#284] [Ideas]
[ReplayGain](https://en.wikipedia.org/wiki/ReplayGain) is a technical standard to normalize loudness of audio(tracks).

It would be great to see loudness equalization across tracks (maybe excluding liveradio and maybe also podcasts) so that there is no unexpected loudness change across tracks.

I personally dont know how it works, like how it is generated or how it needs to be applied, but it seems there is a rust way to generate it: [`replaygain`](https://docs.rs/replaygain/1.0.1/replaygain/).

**Comment by tramhao** [comment]
The info could be saved to database. Need a new config option as well.
